src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -272,6 +281,7 @@ on_exit () {
 trap on_exit EXIT
 
 echo ===========================================================================
+echo '${warningHeader}'
```

Comment:
Can we only add this if there's a header defined? Otherwise it will add a blank line to each output when there's no warning.

Reply:
👍  Updated